### PR TITLE
Added the about us page.

### PR DESCRIPTION
**Pull request by ionutvmi**
Added the about us page..
Added the about us page.
Added the about us page.
Added the about us page.

### CODE REVIEW

File: src/main/java/ro/tefacprogramator/m4/AboutUs.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package ro.tefacprogramator.m4;
```
Comment (ionutvmi) [inline src/main/java/ro/tefacprogramator/m4/AboutUs.java:1]: Example comment.

File: src/main/java/ro/tefacprogramator/m4/AboutUs.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+public class AboutUs {
+
```
Comment (cosminciopor) [inline src/main/java/ro/tefacprogramator/m4/AboutUs.java:4]: Hello world!

**Comment by iusstin** [comment]
comentariu

File: src/main/java/ro/tefacprogramator/m4/AboutUs.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package ro.tefacprogramator.m4;
+
+public class AboutUs {
```
Comment (drugh) [inline src/main/java/ro/tefacprogramator/m4/AboutUs.java:3]: comment

Comment (ionutvmi) [reply src/main/java/ro/tefacprogramator/m4/AboutUs.java:3]: nice comment

File: src/main/java/ro/tefacprogramator/m4/AboutUs.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package ro.tefacprogramator.m4;
+
```
Comment (iusstin) [inline src/main/java/ro/tefacprogramator/m4/AboutUs.java:2]: comentariu

File: src/main/java/ro/tefacprogramator/m4/AboutUs.java (hunk 2 above)
Comment (ionutvmi) [reply src/main/java/ro/tefacprogramator/m4/AboutUs.java:4]: asdasd

- [ ] asdas

- [ ] sadsd

- [ ] asdas

**Comment by ionutvmi** [comment]
some generic comment here.

**Review by ionutvmi — COMMENTED** [review]
approved.

**Review by cosminciopor — APPROVED** [review]
Well done

**Review by iusstin — APPROVED** [review]

**Review by drugh — APPROVED** [review]
i approve

**Pull request merged**